src/lib/QtWidgets/QBoxLayout.ts

```diff
@@ -6,6 +6,7 @@ import { Direction } from '../QtEnums';
 
 export class QBoxLayout extends NodeLayout {
     native: NativeElement;
+    layout: NodeLayout | null;
```

Comment:
you can keep it as an optional 

    `childLayout?: NodeLayout;`

maybe also change the name to childLayout if its okay with you ?